doc/docgen.rst

```diff
@@ -244,7 +244,7 @@ Other Input Formats
 
 The *Nim compiler* also has support for RST (reStructuredText) files with
 the ``rst2html`` and ``rst2tex`` commands. Documents like this one are
-initially written in a dialect of RST which adds support for nim sourcecode
+initially written in a dialect of RST which adds support for nim source code
```

Comment:
I checked - that's the only case where "sourcecode" is used, all other cases use "source code"